ai-service/src/ai_service/embeddings/tests/conftest.py

```
@@ -0,0 +1,25 @@
+"""
+Configuration for embeddings unit tests.
+Initializes the embedding model so tests can run.
+"""
+
+import pytest
+from unittest.mock import patch
+import os
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_embedding_model():
```

Ruff (0.12.2)

Missing return type annotation for public function `setup_embedding_model`

(ANN201)